pom.xml

```diff
@@ -1,124 +1,150 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
-    <groupId>org.obolibrary.robot</groupId>
-    <artifactId>robot</artifactId>
-    <version>0.0.1-SNAPSHOT</version>
-    <packaging>pom</packaging>
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
 
-    <name>robot</name>
-    <description>ROBOT is an OBO Tool</description>
+  <groupId>org.obolibrary.robot</groupId>
+  <artifactId>robot</artifactId>
+  <version>1.0.0-SNAPSHOT</version>
+  <packaging>pom</packaging>
+
+  <name>robot</name>
+  <description>ROBOT is an OBO Tool: A library and command-line tool for working with OWL ontologies, especially Open Biological and Biomedical Ontologes (OBO).</description>
+  <url>https://github.com/ontodev/robot</url>
+  <inceptionYear>2015</inceptionYear>
+
+  <licenses>
+    <license>
+      <name>The 3-Clause BSD License</name>
+      <url>https://raw.githubusercontent.com/ontodev/robot/master/LICENSE.txt</url>
```

Comment:
Is this supposed to be a link to our github content of the 3-clause BSD license? Wouldn't this be a link directly to the origin of the license, namely here: https://opensource.org/licenses/BSD-3-Clause

Reply:
Yeah, I used that link in the first version because that makes more sense to me. But `mvn site` likes the text file better, and embeds it into one of the pages it generates. I think it's OK like this.